### DISCUSSION TITLE
Artemis Kubernetes version supported by Artemis Operator 1.2.0

### DISCUSSION THREAD

**Discussion by lordigon** [arkmq-org/arkmq-org-broker-operator#873] [Q&A] [answered]
Hi all,
I've just downloaded from your repo the content of the 1.2.0 tag and looked inside operator.yaml file that latest and greatest Artemis Kubernetes version supported is 2.32.0.
Looking at your docker repo I saw that version 2.33.0 was already delivered.
Can you tell me when this 2.33.0 version will be supported by the operator?

**Comment by brusdev** [comment, marked as answer]
Hi @lordigon,

the ArtemisCloud Operator [1.2.1](https://github.com/artemiscloud/activemq-artemis-operator/releases/tag/1.2.1) has been released and it supports Apache ActiveMQ Artemis 2.33.0, for further details see https://github.com/artemiscloud/activemq-artemis-operator/releases/tag/1.2.1
